service/src/test/java/org/databiosphere/workspacedataservice/jobexec/QuartzJobTest.java

```diff
@@ -209,7 +209,7 @@ void observationLogsFailure() throws org.quartz.JobExecutionException {
         .hasObservationWithNameEqualTo("wds.job.execute")
         .that()
         .hasHighCardinalityKeyValue("jobId", jobUuid)
-        .hasLowCardinalityKeyValue("jobType", "TestableQuartzJob")
+        .hasLowCardinalityKeyValue("jobType", "jobGroup")
```

Comment:
Super nit: I think a different name would make it more obvious that this is a placeholder value for tests.

```suggestion
        .hasLowCardinalityKeyValue("jobType", "testJobType")
```

Reply:
Fixed with 9f7851bc71801c8d27803e5ffd418fb5f1c0e785